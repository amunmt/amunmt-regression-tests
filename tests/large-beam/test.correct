in Biarritz , however , there is also a need for a little vision .
we , as MEPs , at least also have a duty to encourage them , despite all the odds , to make advances as the messages we get from the public in our countries .
bearing in mind the events of recent times , the issue of petrol prices also seems to me to be particularly remarkable .
the Council is currently examining to what extent these mechanisms are included in Article 7 .
secondly , in terms of transparency for citizens , who are now aware of the rights they have with regard to European law and the creation and transparency of those who do exactly that , European law and European law .
I share his view on the key role that the Commission must also play in order to maintain the common general interest .
I therefore think it is very important that the President of the Euro Group , which we intend to create , should play his role fully in this area .
I personally believe that precisely that is why the Convention method is so useful and that it should therefore be used more frequently in future .
Mr President , Mr President @-@ in @-@ Office of the Council , ladies and gentlemen , thank you for accepting my speech so favourably , and I would also like to thank those who have wisely accused me that I did not have this speech before .
we need to develop enhanced cooperation , make it simpler and more effective in order to achieve this result : enhanced cooperation as the most immediate , the easy way to move towards further integration , the need for which has been confirmed in many good speeches I have heard today .
the debate is closed .
in principle 10 , 11 and 15 are acceptable .
in view of the highly technical nature of this issue , it is quite acceptable to reduce influence by Parliament , especially as the Commission is endeavouring to inform Parliament of its relevant specialist committees on projects to draw up new regulations .
for example , in France , the number of tax revenue varies depending on whether the Finance Ministry &apos;s statistics or national accounts are based .
in order to achieve a rational , effective and safe transport management policy , the priority should be to invest in a public rail network and not to standardise the conditions of competition between the road haulage companies , which are above all to increase their private profit .
such buses will then have access to the country concerned , but they are practically non @-@ existent .
the report supports the approach taken by the Council and the Commission and stresses the need to introduce regulatory measures in a number of areas relating to pensions , cross @-@ border prospectuses and the Community patent .
the biggest problem associated with this new Charter does not concern its actual content , but its legal status .
Mr President , I voted in favour of this Charter , not least because of the influence that our colleague Ingo Friedrich and Professor Herzog have placed on the content of this Charter .
the Charter as a political document is , in our view , a good starting point for this vital dialogue , which must now begin .
this would be a transitional solution which would allow for a binding charter in a certain time .
the Charter goes well behind the European Convention on Human Rights , well behind the national conventions and conventions .
I voted against the possibility of including the Charter of Fundamental Rights before us .
( A5 @-@ 0244 / 2000 ) by Mr Wiersma on Slovakia &#91; COM ( 1999 ) 5@@ 11 - C5 @-@ 0034 / 2000 - 1997 / 23@@ 173 ( COS ) &#93; ;
for this reason , it must be clear that full compliance with the Copenhagen criteria is essential .
in this context , I welcome the work done by the European Parliament in the form of a report on each of the twelve candidate countries in the negotiations and an overall report by Mr Brok , to which I shall refer in particular during what I had to say .
I am thinking , of course , of the common agricultural policy and of regional development policies .
of course , it is not ruled out that the candidate countries should take this opportunity , once again - you have already referred to it , Mr President - to raise the question of the timing of enlargement .
enlargement is a historic achievement .
but success does not only relate to the timing - important though it is .
if they are not implemented everywhere in Europe , they are put at risk everywhere .
chapters will be opened or closed , in line with the actual status of the preparations and the progress of the negotiations .
in the area of the internal market , we must take care to ensure that the transitional periods remain as limited and as limited as possible .
we need political answers to political issues , which must be well communicated .
just imagine how the Kosovo and Bosnia @-@ Herzegovina problems or relations with Serbia would be resolved today at a time of the Cold War .
we should also begin discussions with our own people .
we will be able to support all this and also want to support it .
much of the fears in relation to enlargement stem from exaggerations , but in order to be able to address the populists responsible for this , we must have smart political arguments that convince people .
our group is calling for deadlines to be set at Nice or immediately after the Treaty of Nice in order to end the accession negotiations .
Mr President , our group , the Confederal Group of the European United Left / Nordic Green Left largely shares the considerations and findings contained in Mr Brok &apos;s report on enlargement , and we very much appreciate the efforts of this valuable report .
sixthly , we consider it a mistake to link enlargement in practice with the previous integration of these countries into NATO .
like many others this morning , I very much welcomed the statements made by President Prodi ; I have taken note of the statements made by Minister Moscovici , who has also pointed to us , albeit diplomatically , to the fact that the current situation is not particularly optimistic with regard to the possible results of the next two or three months .
I therefore ask the President @-@ in @-@ Office and the Commissioner to take due account of the length of their contributions .
I would draw the conclusion that this needs to be discussed more , but also in an appropriate and better way .
however , irrespective of this , I do not think it is inappropriate to raise this issue .
I do not want to let slip into the trap that has existed for many years in other areas of foreign aid .
however , the fact is that , today , 3 October 2000 , I cannot tell you any country in the accession process when it will be ready for accession .
I can also tell them the reason why , because these people wanted to be on the right side , they are firmly anchored in the family of democratic nations .
thank you for that clarification , Commissioner .
in my view , she would not have met her brief and would have missed her point .
now we can secure common values and benefits .
imagine if , in your own country , a fundamental , economic and social transformation has taken place in such a short space of time .
it is still hot air , Mr Verheugen , and there is still no sign of the concrete project promoters .
today , the macroeconomic situation has stabilised .
this issue needs to be made a special effort and , in my opinion , cannot be dealt with simply as a marginal issue for the accessions of Lithuania and Poland .
for example , if we consider the possibility that citizens , the workers of the candidate countries can move freely in the European Union from the outset , if we want to delay the recognition of this right to the fundamental principles of the Union from the East , this means further erosion of support for the difficult reforms needed for accession .
I am pleased that the report was adopted unanimously in committee on 14 September .
my colleagues in the Committee on Foreign Affairs , Human Rights , Common Security and Defence Policy supported me in it - that the Czechs continue to apply the Copenhagen criteria , and I would like the Czechs themselves to check their own legal system when it has to be , side by side , to find out whether there are some things that are discriminatory .
and now on the good news , because they are putting my critical comments into account in the background .
Mr President , ladies and gentlemen , I think I can say that , having discussed and fleshed out in the Committee on Foreign Affairs and adopted unanimously in the Committee on Foreign Affairs , this report gives a clear description and analysis of the present phase of Hungary &apos;s journey towards accession in two respects .
we need to address this issue seriously , but within a global framework which includes not only Hungary but also the other candidate countries which have this obsolete and unsafe technology .
the suspicion of corruption is an extremely destructive force and must be stopped as a matter of urgency .
thanks to the active support of Commissioner Verheugen , the World Bank and the United Nations , the Romanian Government has given us the opportunity to introduce exemplary principles and best practices in the region .
this is a candidate country in which a third of the territory has been occupied by the Turkish army since 1974 and in which the basic principles of the European Union do not apply throughout its territory because of a particularly impenetrable demarcation line .
Europe must become a responsible player in resolving the Cyprus problem .
for Malta , it is essential to provide financial support in the framework of the pre @-@ accession strategy , as well as an awareness that the aid to Malta should comply with the rules that apply to other candidate countries .
once the relevant conditions are met , neither the candidate countries nor the countries in the eurozone present have any reason to oppose the accession of candidate countries to the third stage of economic and monetary union .
if you like , I am a representative of a region that ten years ago , represented a piece of enlargement to the east , I lived beyond the Iron Curtain .
in the interests of a common information area , we must work together to make great efforts in this area .
there is no doubt that there are still differences in the area of technology between East and West : 4 Internet connections per 1 000 inhabitants of Central and Eastern Europe are 16 of us .
in this situation , the German Democratic Republic was given massive financial assistance with West Germany , £ 150 billion a year .
in the next few years , in Eastern Europe , there is a threat to the depopulation of villages in rural areas and a massive purchase of agricultural land to mechanical , productive holdings .
we are drawing up a compendium of directives and regulations on issues such as pollution , the emission of radioactive and other dangerous substances , waste disposal , water , air and soil quality , food safety and novel food and product liability , which are important for health .
in my opinion , therefore , my opinion is not just about what the candidate countries should do , but also of what the European Union must do .
but now to the right signals .
we must make sure that what is present in central and eastern Europe is not replaced by road transport , now it is not replaced by road transport and , afterwards again , with a great deal of effort to put things right .
this &apos; no &apos; vote will be analysed very closely in the coming months .
we are working through the text and tried , on a number of points , to make some penalties .
there should be no doubt as to the value , including the full @-@ material value of this enlargement in the public .
if a long transitional period is required in certain areas , then we should accept them .
first , the democratic desire of the states and peoples concerned to join the EU is to be respected .
they are mistaken because , with 80 000 pages of directives and regulations , the European Union will not create prosperity .
Denmark is demonstrating the increasing reservations that are being shown by our people and the recent rejection of the euro by Denmark .
the opinion of the Brok report , for which I had the responsibility and which was supported by the Committee on Budgets , comes to the same conclusion .
this is not the best time to address foreign policy issues and the mutual accusations of the different parties are sometimes exacerbated and manipulated .
if the EU rules were split into two , we could , on the one hand , safeguard the level of integration achieved in the present Member States and , on the other , promote the deepening of their cooperation .
I visited Romania at the end of July to see for myself the functioning of one of the largest orphanages in northern Romania .
for example , we ask the rapporteur , the Commission and the Council to complement the agricultural policy part of the fisheries policy , for example , by referring to the &apos; agricultural aspects &apos; of the enlargement and fisheries policy aspects of enlargement .
better later and well than earlier and bad .
thanks to the close relations between Cyprus and the people of the Eastern Mediterranean , the accession of the island will increase the presence of Europe in this very important area .
I support the rapporteur &apos;s view that Malta should be included in the first round of enlargement and that the negotiations should be completed at the end of 2001 .
it is our responsibility , too , to communicate enlargement through speeches and through action .
in this context , it is important to create a positive picture of enlargement .
the point made by Mr Verheugen , a referendum on enlargement , has been thrown out , and the reason is obvious : the owners will not be able to decide on it .
in view of the double challenge that Europe should be able to reconcile itself , and the institutional reform intended to strengthen democracy and efficiency in its institutions , the European Union - as so often - is at a significant turning point in its history .
I would like to focus my remarks on the Council , which is unfortunately absent , but I would like to express a number of expectations and concerns .
this dialogue would also help to create the necessary administrative capacity for the effective governance of the country , by creating the structures that are not yet available today and the levels of corruption being reduced .
Malta , like the other smaller Member States , should be represented by at least six Members .
nor can we assume that some people are , of course , the first candidate and could not , without him , begin enlargement at all .
we can only hope that the great efforts that the government has just made recently will be reflected in the Commission &apos;s next report .
