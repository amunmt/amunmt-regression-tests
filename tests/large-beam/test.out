in Biarritz , however , there is also a need for a little vision .
we , as MEPs , at least also have a duty to encourage them , despite all the odds , to make advances as the messages we get from the public in our countries .
bearing in mind the events of recent times , the issue of petrol prices also seems to me to be particularly remarkable .
the Council is currently examining to what extent these mechanisms are included in Article 7 .
